dss/error.py

```diff
@@ -81,7 +98,10 @@ def wrapper(*args, **kwargs):
             code = "read_only"
             title = "The DSS is currently read-only"
             stacktrace = ""
-            headers = {'Retry-After': 600}
+            headers = {'Retry-After': '600'}
```

Comment:
It is not an error to use an integer.

Reply:
I wanted to keep it consistent.

Reply:
It also errors if the header is added in `app.py` because swagger validates it:
```
ERROR:dss:Internal Error for <function get_chalice_app.<locals>.dispatch at 0x7fcb972786a8>
Traceback (most recent call last):
  File "/home/quokka/git/data-store/v3nv/lib/python3.6/site-packages/chalice/app.py", line 845, in _get_view_function_response
    self._validate_response(response)
  File "/home/quokka/git/data-store/v3nv/lib/python3.6/site-packages/chalice/app.py", line 873, in _validate_response
    if '\n' in value:
TypeError: argument of type 'int' is not iterable
```

I'd like to keep them all `str` rather than some `str` and some not.  What do you think?

Reply:
str is fine